heron/spi/src/java/com/twitter/heron/spi/common/Context.java

```diff
@@ -34,6 +34,10 @@ public static String environ(Config cfg) {
     return cfg.getStringValue(ConfigKeys.get("ENVIRON"));
   }
 
+  public static String dryRun(Config cfg) {
```

Comment:
actually, the config value you got is the dry-run format that's going to be used.